scrypto/src/engine/scrypto_env.rs

```diff
@@ -449,6 +449,13 @@ impl ClientTransactionRuntimeApi<ClientApiError> for ScryptoEnv {
 
         scrypto_decode(&actor).map_err(ClientApiError::DecodeError)
     }
+
+    fn panic(&mut self, message: String) -> Result<(), ClientApiError> {
+        unsafe {
```

Comment:
Just a thought - but when we implement limits for this, we should probably limit the message length here too, to truncate things without exploding upstream.

Reply:
I agree